peps/pep-0770.rst

```diff
@@ -139,6 +139,29 @@ These decisions mean this PEP is capable of supporting any SBOM standard
 and does not favor one over the other, instead deferring the decision to
 producing projects and tools and consuming user tooling.
 
+Adding data to Python packages without new metadata versions
+------------------------------------------------------------
+
+The rollout of a new metadata version and field requires that many different
+projects and teams need to adopt the metadata version in sequence to avoid
+widespread breakage. This effect usually means a substantial delay in how
+quickly users and tools can start using new packaging features.
+
+For example, a single metadata version bump requires
+updates to PyPI, various ``pyproject.toml`` parsing and schema projects,
+the ``packaging`` library, wait for releases, then ``pip`` and other installers
+need to bundle the changes to ``packaging`` and release, then build backends can
+begin emitting the new metadata version, again wait for releases, and only then
+can projects begin using the new features. Even with this careful approach it's
+not guaranteed that tools won't break on new metadata versions and fields.
+
+To avoid this delay, this PEP proposes a new mechanism ``[additional-files]``
+to safely add data to a Python package through a registry of reserved names that
```

Comment:
This might be more suited to a discuss.p.o comment, but `[additional-files]` feels a somewhat out of place to me as a name ("additional files for what?").

Something more self-describing that it is for built (or source) distributions, rather than for the source tree / development work might be nicer. For example `[dependency-groups]` is a top-level table that has no impact on distribution packages, so it isn't the case that top-level tables always relate to distribution packages.

The best I can come up with is `[distribution-files]` but it isn't great!

Reply:
The `additional-files` name was just my throwaway suggestion. I honestly expected way more bikeshedding than it got. But yes, this should probably be raised on Discourse.

Reply:
See https://discuss.python.org/t/76308/76